common/plugins/org.jboss.tools.common.jaxb/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,18 @@
+Manifest-Version: 1.0
+Bundle-Name: %Bundle-Name.0
+Bundle-Vendor: %providerName
+Bundle-SymbolicName: org.jboss.tools.common.jaxb;singleton:=true
+Bundle-Localization: plugin
+Bundle-Version: 3.10.2.qualifier
```

Comment:
wouldn't it be preferable to have the bundle match the version of the jaxb within it?